lte/cloud/go/services/lte/obsidian/models/defaults.go

```diff
@@ -79,5 +79,6 @@ func NewDefaultEnodebStatus() *EnodebState {
 		MmeConnected:     swag.Bool(true),
 		FsmState:         swag.String("TEST"),
 		IPAddress:        "192.168.0.1",
+		UesConnected:     5,
```

Comment:
This file should be in a separate `test_models` package. Since it's kind of a big change we don't have to do it here. Instead, can you just add a doc comment to the three functions in this file that they're only supposed to be used in tests